docs/windows-getting-started.md

```diff
@@ -0,0 +1,28 @@
+# Windows: Getting Started
+
+Currently, **runwasi** depends on POSIX environment (i.e., because it has to wire up networking and rootfs mounts). Therefore, to run it on Windows, we recommend utilizing the Windows Subsystem for Linux (WSL).
```

Comment:
Not just POSIX, this is using Linux specific features.

Reply:
I just changed this!